src/CounterInterceptor/MyActivities.cs

```diff
@@ -0,0 +1,17 @@
+namespace TemporalioSamples.CounterInterceptor;
+
+using System.Diagnostics;
+using Temporalio.Activities;
+
+public class MyActivities
+{
+    [Activity]
+    public string SayHello(string name, string title) =>
+        "Hello " + title + " " + name;
+
+    [Activity]
+    public string SayGoodBye(string name, string title)
```

Comment:
You are inconsistent with your single-line methods, this should be `=>` like the one just above it

Reply:
Fixed